Source/Plugins/OrientationAnalysis/OrientationAnalysisFilters/ReadCtfData.cpp

```diff
@@ -315,8 +312,8 @@ void ReadCtfData::readDataFile(CtfReader* reader, DataContainer::Pointer m, QVec
       if(err < 0)
       {
         setErrorCondition(err);
-        notifyErrorMessage(getHumanLabel(), "CtfReader could not read the .ctf file header.", getErrorCondition());
-        notifyErrorMessage(getHumanLabel(), reader->getErrorMessage(), err);
+        notifyErrorMessage("", "CtfReader could not read the .ctf file header.", -997);
```

Comment:
-997 is propagating.  We still have the `setErrorCondition(..)` here.

Reply:
Don't ignore the error message from lower classes. In this instance (And similar instances of EBSD Readers and other readers) the lower class has a very detailed error message and error code that is getting ignored (the message in this case).